client/components/Settings.tsx

```diff
@@ -93,20 +113,20 @@ const Settings = () => {
       
 
         <View className='w-full items-center text-white'>
-        <Text className='text-white text-2xl self-start'>Update Reading Level</Text>
-        <View className='bg-grey w-full rounded-full px-4 py-2 border border-green text-white'> 
-          <Picker
-            className='w-full text-white grey'
-            selectedValue={selectedProfile?.readingLevel}
-            onValueChange={handleReadingLevelChange}
-            dropdownIconColor='#ffffff'      
-            selectionColor='#ffffff'      
-          >
-            {Object.keys(readingLevelOptions).map((level) => (
-              <Picker.Item key={level} label={`${level}`} value={level} style={{backgroundColor : "#333333", color: '#ffffff'}} />  
-            ))}
-          </Picker>
-        </View>
+          <Text className='text-white text-2xl self-start'>Update Reading Level</Text>
```

Comment:
This isn't current dev. I have removed the Picker into it's own file, please update